src/pages/GiftAid/utils/validateForm.js

```diff
@@ -70,7 +70,7 @@ const validateTransactionId = (donationID) => new RegExp(transactionIdPattern).t
 const getValidation = (validation) => {
   let validity = true;
   Object.keys(validation).map((key) => {
-    if (validation[key].valid !== true && validation[key].valid !== null ) {
+    if (validation[key].valid !== true && validation[key].valid !== "" && validation[key].valid !== null) {
```

Comment:
should be `if (validation[key].valid !== true || validation[key].valid === "" || validation[key].valid === null) {
      validity = false;
    }`

Reply:
or actually checking for !== true should already cover the other options
I see the code was already checking `if (validation[key].valid !== true && validation[key].valid !== ''){validity = false; } ` which is a bit odd.